README.md

```diff
@@ -28,7 +28,20 @@ OR:
 To Deploy
 --------------
 
-Ssh into the Visuals server. Cd into the gudocs repo and git pull master. 
+*This deployment process is difficult to use and should be upgraded to RiffRaff asap*
+
+Ssh into the Visuals server (you need to have your github keys added to access it). Cd into the gudocs repo and git pull master. You may need to run commands like `npm install` if you have eg. installed any different dependencies.
```

Comment:
Does someone who already has access add the keys for you?

Reply:
yeah that's right. I can add that in